File: frontend/src/app/nutricionista/getAllNutricionisteResponse.ts
import {GetAllKorisnickiNalogResponseKorisnickiNalog} from "../korisnicki-nalog/getAllKorisnickiNalogResponse";

export interface GetAllNutricionisteResponse {
    nutricionisti: GetAllNutricionistaResponseNutricionista[];
}
export interface GetAllNutricionistaResponseNutricionista {
    zaposlenikId:number
    imePrezime: string;
    jmbg: string;
    datumRodjenja: Date;
    datumZaposlenja: Date;
    nalogId: number|null;
    poslovnaPozicijaId: number;
    oblastNutricionizma:string;
    nutricionistickiCentar:string;
}

File: DomZaStaraLicaApi/Endpoints/Nutricionista/GetAllNutricioniste/GetAllNutricionisteResponse.cs
namespace DomZaStaraLicaApi.Endpoints.Nutricionista.GetAllNutricioniste
{
    public class GetAllNutricionisteResponse
    {
        public List<GetAllNutricionisteResponseNutricioniste> Nutricionisti { get; set; }
    }
    public class GetAllNutricionisteResponseNutricioniste
    {
        public int ZaposlenikId { get; set; }
        public string ImePrezime { get; set; }
        public string JMBG { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public int? NalogId { get; set; }
        public int PoslovnaPozicijaId { get; set; }
        public string NutricionistickiCentar { get; set; }
        public string OblastNutricionizma { get; set; }
    }
}
